Mathlib/RingTheory/Polynomial/Chebyshev.lean

```diff
@@ -299,6 +499,23 @@ theorem mul_T (m k : ℤ) : 2 * T R m * T R k = T R (m + k) + T R (m - k) := by
     have h₃ := T_add_two R (-k - 1)
     linear_combination (norm := ring_nf) 2 * T R m * h₃ - h₂ - h₁ - ih2 + 2 * (X : R[X]) * ih1
 
+/-- The product of two Chebyshev `C` polynomials is the sum of two other Chebyshev `C` polynomials.
+-/
+theorem mul_C (m k : ℤ) : C R m * C R k = C R (m + k) + C R (m - k) := by
```

Comment:
```suggestion
theorem C_mul_C (m k : ℤ) : C R m * C R k = C R (m + k) + C R (m - k) := by
```

Reply:
And don't forget to update the documentation at the top of the file